frontend/kick-covid-19/src/pages/Blog.js

```diff
@@ -21,11 +21,14 @@ class Blog extends React.Component {
   }
 
   render() {
+    console.log(
+      `[DEBUG] this.state.posts = ${JSON.stringify(this.state.posts, null, 2)}`
+    );
     return (
       <div className="row">
-        {this.state.posts.map((body, posts, id) => (
-          <Link key={posts.id} to={`/posts/${posts.id}`}>
-            <Post body={body} key={id} />
+        {this.state.posts.map(({ _id, body, answers }) => (
+          <Link key={_id} to={`/posts/${_id}`}>
+            <Post body={body} />
```

Comment:
also, I feel like you'll want to add a title to your posts schema (e.g. a question/post would most likely have a title and a body) based on quora/stack overflow model?  same with blog articles where there is typically a title and short description etc.. that is rendered in the list view

Reply:
and then in your detail view you would show the body and answers